### DISCUSSION TITLE
Downloading Annotations in a separate environment

### DISCUSSION THREAD

**Discussion by digs1998** [DigitalSlideArchive/digital_slide_archive#321] [General]
Hi, I was wondering if is it possible for me to download the files directly in the VM, from where it is being run? I get the port from docker forwarded to my local device, and accessing the browser from there, I am unable to save files in the VM.

**Comment by manthey** [comment]
I'm not sure what you are asking.  If you are running the default deployment, it is in a set of docker containers and the files are either in the default assetstore (which is just the devops/dsa/assetstore folder) or imported from somewhere on your filesystem.  Downloads are from the system's API via whatever url it needs relative to your location.  You can access files in the assetstore directly, but they are stored by hash not name which makes this confusing.  You can also use girder-client for command-line access to downloading files.
